openedx/openedx-platform
# feat: add course listing alongside learner dashboard

## Description

- Add stub files for new learner dashboard backend under the currently existing `learner_dashboard` djangoapp.
- Rename/refactor some existing files from the Programs Dashboard to more clearly delineate the two projects that both exist under this common djangoapp.

**Note:** ideally, the programs dashboard would break out into a separate project but this was a compromise to allow us to use the most appropriate app name for our project w/out having to do lots of other code shifting.

## Supporting information

JIRA: [AU-698](https://2u-internal.atlassian.net/browse/AU-698)
FYI: @openedx/content-aurora 

## Testing instructions

With a logged in user, exercise the following endpoint to verify routing:

```
GET {LMS}/dashboard/learner
```

Should return empty array for testing.
